### ISSUE TITLE
Livechat window not showing up

### ISSUE THREAD

**Issue by jemus42** [RocketChat/WordPressPlugin#14]
- WordPress 4.7.3
- Rocket.Chat 0.54.2

I have done the following:

- Set up Rocket.Chat and make sure everything works fine
- Move it behind an nginx reverse proxy which I verified does *not* set the `X-Frame-Options DENY` header
- Install the wordpress plugin on an otherwise fully functioning wordpress install
- Enabled the plugin in settings 
- Entered the Rocket.Chat URL
- Tried both `my.domain.tld` and `my.domain.tld/livechat` as URL
- Verified that livechat works _in principle_ by pasting the embed javascript into an empty `.html` file and opened it in my browser locally
- Moved that same file to a webserver and opened it in my browser, once again it worked just fine

…and at last, I just copypasted the install code in my wordpress' raw html, and now it works just fine.

TL;DR I think the plugin might need an update.

**Comment by HLFH** [comment]
Thanks. I disabled the nginx line `add_header X-Frame-Options "SAMEORIGIN";`

**Comment by saschafoerster** [comment]
+1

**Comment by mbanusic** [comment]
New code has been updated in e6ac2556f9bef0b50fd4a88d75bf8879e5e71b4a and has also been uploaded to wordpress.org

**Comment by plmorpheus** [comment]
I think that plugin needs update once again because it still not works. I tried run code provided by Rocket from html file located on my desktop and it didn't appear. In console I see "Refused to display 'https://my.rocket.chat.url/livechat' in a frame because it set 'X-Frame-Options' to 'sameorigin'." I have docker version of RC, 3.4.2.
